src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesNotShown.jsx

```diff
@@ -105,7 +104,10 @@ export default function FacilitiesNotShown({
               </li>
             ))}
           </ul>
+          <strong>What you can do</strong>
```

Comment:
Seems like this should be inside `<p>` below, so that it doesn't look like it's part of the list above.

Reply:
Fixed 
![image](https://user-images.githubusercontent.com/8315447/102938651-b6e39880-447a-11eb-9a7c-26c5673b5c3a.png)
![image](https://user-images.githubusercontent.com/8315447/102938667-bf3bd380-447a-11eb-8a60-41c0a849e9ae.png)